pkgs/tools/bootloaders/limine/default.nix

```diff
@@ -0,0 +1,31 @@
+{ pkgs, lib, fetchurl, llvm } :
+
+pkgs.stdenv.mkDerivation rec {
+  name = "limine";
+  version = "4.20221006.1";
+  src = fetchurl {
+    url = "https://github.com/limine-bootloader/limine/releases/download/v${version}/limine-${version}.tar.xz";
+    sha256 = "sha256-BZBRhc1h6C2CMrppulz+zHcRi3Pe52+sRwQkXeMG8hk=";
+  };
+  nativeBuildInputs = [
+    pkgs.autoconf
+    pkgs.automake
+    pkgs.nasm
+    llvm.bintools
+    llvm.clang
```

Comment:
You can use `clangStdenv` instead of having a package that depends on both gcc (from stdenv) and clang

Reply:
Done.